tests/src/observation/test_grdc.py

```diff
@@ -80,33 +79,36 @@ def expected_results(tmp_path, sample_grdc_file):
         "river_name": "SOME RIVER",
         "station_name": "SOME",
         "time_series": "2000-01 - 2000-01",
-        "units": "m³/s",
         "UserEndTime": "2000-02-01T00:00Z",
         "UserStartTime": "2000-01-01T00:00Z",
         "nrMissingData": 1,
     }
-    return data, metadata
+    var_metadata = {
+        "long_name": "MEAN DAILY DISCHARGE (Q)",
+        "units": "m³/s",
```

Comment:
The superscript can sometimes be problematic... Usually it's denoted `m3 s-1`. [for example](https://github.com/PCMDI/cmip6-cmor-tables/blob/11312205dd53c504a51276af5292fb367ab88f2f/Tables/CMIP6_Eday.json#L1425)
```suggestion
        "units": "m3 s-1",
```

Reply:
Also, the usgs data has as units "cubic meters per second"

Reply:
the netcdf downloaded from grdc has 'm3/s' so using that